bootstrap/kubeadm/main.go

```diff
@@ -54,18 +54,17 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
-// nolint:gocyclo
-func main() {
-	var (
-		metricsAddr              string
-		enableLeaderElection     bool
-		watchNamespace           string
-		profilerAddress          string
-		kubeadmConfigConcurrency int
-		syncPeriod               time.Duration
-		webhookPort              int
-	)
+var (
+	metricsAddr              string
+	enableLeaderElection     bool
+	watchNamespace           string
+	profilerAddress          string
+	kubeadmConfigConcurrency int
+	syncPeriod               time.Duration
+	webhookPort              int
```

Comment:
Any particular reason to make these global here? I'd rather see us avoid that if at all possible. Should we have a struct we pass between main and the `setup` methods instead?